### DISCUSSION TITLE
rendered SINR image

### DISCUSSION THREAD

**Discussion by lufeiaixuexi** [NVlabs/sionna#559] [Q&A] [answered]
Hello;
I would like to know if it is possible to obtain a rendered SINR image in addition to Coverage Map

**Comment by jhoydis** [comment, marked as answer]
Hi @lufeiaixuexi,

Please have a look at this [tutorial](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Coverage_Map.html).

**Reply by lufeiaixuexi** [reply]
ok, Thank you!
